src/components/auth/SignUp.js

```diff
@@ -66,64 +67,77 @@ const SignUp = (props) => {
+                            <img src={GoogleLogo} alt="Google Logo" className="absolute left-0 ml-5"/>
+                            <p className="text-xl">Login with Google</p>
+                        </button>
+
+                        <p className="w-full text-center border-b leading-middle my-10 inline-block border-black">
+                            <small className="bg-white px-2 text-base">Or</small>
+                        </p>
+
+                        {/* <div className="flex flex-row"> */}
+                            <div className="py-2 pb-2">
+                                <label htmlFor="email" className="text-sm block text-xl">First Name</label>
                                 <input
-                                className="textshadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline border-blue-300"
-                                placeholder="Last Name"
-                                type="text"
-                                name="last_name"
-                                value={credentials.last_name}
-                                onChange={handleChange}
-                            />
-                        </div>
-                        <div className="px-4 pb-4">
-                            <label htmlFor="email" className="text-sm block font-bold  pb-2">Email</label>
+                                    className="textshadow appearance-none border rounded w-full py-4 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline border-gray-500"
+                                    type="text"
+                                    name="first_name"
+                                    value={credentials.first_name}
+                                    onChange={handleChange}
+                                />
+                            </div>
+                            <div className="py-2 pb-2">
+                                <label htmlFor="email" className="text-sm block text-xl">Last Name</label>
                                 <input
-                                className="textshadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline border-blue-300"
-                                placeholder="email"
+                                    className="textshadow appearance-none border rounded w-full py-4 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline border-gray-500"
+                                    type="text"
+                                    name="last_name"
+                                    value={credentials.last_name}
+                                    onChange={handleChange}
+                                />
+                            </div>
+                        {/* </div> */}
```

Comment:
Dangling comment?

Reply:
I commented out the div portions that caused the First/Last Name to sit side by side, commenting them out have them set in a column as per requested by the UX team.